.github/workflows/bundle-desktop.yml

```diff
@@ -201,21 +201,5 @@ jobs:
             echo "App did not stay open. Possible crash or startup error."
             exit 1
           fi
-          LOGFILE="$HOME/Library/Application Support/Goose/logs/main.log"
```

Comment:
why was this removed?

Reply:
because once I removed react router - this log is not generated anymore, so no way to test this reliably